site/src/pages/UsersPage/UsersTable/UsersTableBody.tsx

```diff
@@ -155,7 +155,7 @@ export const UsersTableBody: FC<
                   : sortRoles(user.roles);
 
               return (
-                <TableRow key={user.id}>
+                <TableRow key={user.id} data-testid={`user-${user.id}`}>
```

Comment:
nit: I don't remember if `data-testid` must be a unique id across the entire page, but if so then `user-<id>` might be easy to accidentally appear somewhere else.

Reply:
I've checked already and this `data-testid` should be unique. This practice is already used in many other places e.g. workspaces table rows, workspace build table rows.